thon3.10/dist-packages/pika/connection.py", line 656, in __init__
    raise TypeError('unexpected kwargs: %r' % (kwargs,))
TypeError: unexpected kwargs: {'heartbeat_interval': 600}
